### PR TITLE
 Feat/mkt 7554 : Added branch support for getContentTypes()

### PR DESCRIPTION
**Pull request by MaitriGurey1**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=h92Aajt)

### CODE REVIEW

File: src/stack/index.ts [hunk 1]
```diff
@@ -144,10 +144,18 @@ class Stack {
    * @param {Object} params Optional parameters for the GET call
    * @return {Object} A promise object which will be resolved with details of the content type.
    */
-  getContentTypes(query = {}, params: { [key: string]: any } = {}): Promise<{ [key: string]: any }> {
+  getContentTypes(branch?: string, query = {}, params: { [key: string]: any } = {}): Promise<{ [key: string]: any }> {
```
Comment (rijil-tr) [inline src/stack/index.ts:147]: Why not just accept the branch parameter in params? So there won't be any change in signature but works fine when user started passing branch.
```
const { branch, ... optionParams } = params;
```

Comment (MaitriGurey1) [reply src/stack/index.ts]: done

File: src/stack/index.ts [hunk 2]
```diff
@@ -145,9 +145,17 @@ class Stack {
    * @return {Object} A promise object which will be resolved with details of the content type.
    */
   getContentTypes(query = {}, params: { [key: string]: any } = {}): Promise<{ [key: string]: any }> {
-    const optionParams = params;
+    const { branch, ... optionParams } = params;
```
Comment (rijil-tr) [inline src/stack/index.ts:148]: @MaitriGurey1 Why there is an extra space after ... ?

**Review by rijil-tr — APPROVED** [review]
LGTM!

**Pull request merged**